### PR TITLE
Domain register

### PR DESCRIPTION
**Pull request by MaskDuck**
Register of domain maskduck.is-a.dev

### CODE REVIEW

**Comment by continental-beauty-msbacdau** [comment]
hi, when will my domain become accepted?

**Comment by MaskDuck** [comment]
@TheBotlyNoob

**Comment by MaskDuck** [comment]
@TheBotlyNoob @TheBotlyNoob-ALT @phenax please see this plz

**Review by TheBotlyNoob — APPROVED** [review]

File: domains/maskduck.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "MaskDuck's personal website!",
+  "repo": "https://github.com/continental-beauty-msbacdau/continental-beauty-msbacdau.github.io",
+  "owner": {
+    "github": "continental-beauty-msbacdau",
```
Comment (TheBotlyNoob) [inline domains/maskduck.json:5]: change the `github` property to `username`

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

File: domains/maskduck.json (hunk 1 above)
Comment (MaskDuck) [reply domains/maskduck.json:5]: @TheBotlyNoob, you mean change `github` into `username`?

**Review by TheBotlyNoob — DISMISSED** [review]

**Comment by MaskDuck** [comment]
@TheBotlyNoob when will it merged?

**Comment by BOssTDiscord** [comment]
> @TheBotlyNoob when will it merged?

7-11 days

**Comment by MaskDuck** [comment]
its the 8th day and my pr didnt merged yet :(

**Comment by MaskDuck** [comment]
its 10th day :(

**Comment by MaskDuck** [comment]
@TheBotlyNoob @phenax @TheBotlyNoob-ALT

**Comment by MaskDuck** [comment]
@TheBotlyNoob

**Comment by MaskDuck** [comment]
@TheBotlyNoob @TheBotlyNoob-ALT @phenax

**Comment by MaskDuck** [comment]
@is-a-dev

**Comment by MaskDuck** [comment]
@is-a-dev

**Comment by MaskDuck** [comment]
@TheBotlyNoob @phenax

**Comment by MaskDuck** [comment]
it is open for almost 1 mo

**Comment by MaskDuck** [comment]
it is opened for over a month now @TheBotlyNoob @is-a-dev

**Review by phenax — APPROVED** [review]
Sorry for the delay. (#1529)

**Pull request merged**